tests/phpunit/integration/Core/Tags/Guards/Tag_Production_GuardTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Tag_Production_GuardTest
+ *
+ * @package   Google\Site_Kit\Tests\Core\Tags\Guards
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Tests\Core\Tags\Guards;
+
+	use Google\Site_Kit\Core\Tags\Guards\Tag_Production_Guard;
+	use Google\Site_Kit\Tests\TestCase;
+
+class Tag_Production_GuardTest extends TestCase {
+
+	public function test_can_activate_on_older_environments() {
+		// Pre WP-5.5.0
+		if ( ! function_exists( 'wp_get_environment_type' ) ) {
+			$this->markTestSkipped( 'Missing wp_get_environment_type() function.' );
+		}
+		$tagproduction = new Tag_Production_Guard();
+		$this->assertTrue( $tagproduction->can_activate() );
+	}
```

Comment:
I think if we want to test it for older versions of WP, we need to skip this test if the function exists, right?

Reply:
Maybe let's rename the method name to be something like `test_can_activate_on_older_versions`?

Reply:
https://github.com/google/site-kit-wp/pull/3734/commits/a81ec2efe6837400ef41a3b32d8f2d40b0dbb31a

Reply:
Updated commenting for both in https://github.com/google/site-kit-wp/pull/3734/commits/267859182074ef4640ca2902182371aeb6596854

First test `test_can_activate_on_post_5_5_version()` is for post 5.5.0 scenarios, and thus skips if `wp_get_environment_type()` is not defined.
Second test `test_can_activate_on_older_versions()` is for pre 5.5.0 scenarios, and thus skips if `wp_get_environment_type()` is defined.